app/src/main/java/pl/futurecollars/invoicing/db/memory/InMemoryDatabase.java

```diff
@@ -0,0 +1,49 @@
+package pl.futurecollars.invoicing.db.memory;
+
+import pl.futurecollars.invoicing.db.Database;
+import pl.futurecollars.invoicing.model.Invoice;
+import java.util.*;
+
+public class InMemoryDatabase implements Database {
+
+
+    public final Map<UUID, Invoice> database = new HashMap<>();
```

Comment:
This field should be private - we don't want to have access to this Map outside of our interface